content/docs/operating/configuration.md

```diff
@@ -278,14 +278,17 @@ CAUTION: Kubernetes SD is in beta: breaking changes to configuration are still
 likely in future releases.
 
 Kubernetes SD configurations allow retrieving scrape targets from
-[Kubernetes'](http://kubernetes.io/) REST API. By default, this discovers nodes
-and appropriately annotated services so that metrics from both cluster
-components and deployed applications can be scraped. This will create multiple
-target groups: one for all nodes with each node as a target; and one for each
-service containing each service endpoint as a target.
+[Kubernetes'](http://kubernetes.io/) REST API. By default, this discovers
+masters, nodes and appropriately annotated services so that metrics from both
```

Comment:
Ah the Oxford comma... https://stancarey.files.wordpress.com/2014/09/socratic-tweet-oxford-comma-on-mandela-800-year-old-demigod-and-dildo-collector.jpg